packages/styles/scss/patterns/leadspace-centered/index.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../globals/imports';
+@import '@carbon/ibmdotcom-styles/scss/components/buttongroup/buttongroup.scss';
+@import './leadspace-centered';
+
+.#{$prefix}--leadspace--g100 {
+  @include carbon--theme($carbon--theme--g100) {
+    @include leadspace-centered;
+  }
+}
```

Comment:
@annawen1 after this afternoon's discussion, let's move the theme modifier into the pattern level